docs/protocol/extension_generate.md

```diff
@@ -169,7 +169,7 @@ the HTTP body.
 
 A failed generate request must be indicated by an HTTP error status
 (typically 400). The HTTP body must contain the
-*$generate_error_response* object.
+`$generate_error_response` object.
```

Comment:
ditto:
![image](https://github.com/triton-inference-server/server/assets/124622579/94703b1e-d916-4964-bdd9-610cc1d8eecd)

Reply:
Actually, no worries, let's keep this for consistency